test/ex_doc/refs_test.exs

```diff
@@ -10,6 +10,10 @@ defmodule ExDoc.RefsTest do
     def foo(), do: :ok
   end
 
+  setup do
+    ExDoc.Refs.clear()
+  end
```

Comment:
this still can be fixed. Let me know so I can submit a PR

Reply:
Clearing will cause issues though because we can erase something that has just been defined in another test. Ideally we shouldn't ever clear.